### PR TITLE
[1062] Trial Pages Redesign 

### PR DESCRIPTION
**Pull request by thujcak**
**Changes proposed in this Pull Request**
added new page-templates, assets, and styles

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#1062

### CODE REVIEW

File: assets/images/trial_badges.svg
Comment (jremen) [inline assets/images/trial_badges.svg:1]: This one is better to save as PNG with 8 bit transparency

File: assets/images/trial_sidebar_bg.png
Comment (jremen) [inline assets/images/trial_sidebar_bg.png:1]: use ImageOptim app to attempt to reduce filesize.

File: assets/styles/layouts/_FullScreen.scss [hunk 1]
```diff
@@ -235,17 +234,17 @@ body.page-template-template-demo {
...
-			@include posZero;
-			left: -25%;
-			right: 25%;
-			position: absolute;
-			content: "";
-			z-index: 1;
-			background: $lightest-gray url(../images/trial_bg.svg) center center no-repeat;
-			background-size: cover;
-		}
+		// &::after {
```
Comment (jremen) [inline assets/styles/layouts/_FullScreen.scss:237]: remove commented out code

File: assets/styles/layouts/_FullScreen.scss [hunk 2]
```diff
@@ -124,9 +123,9 @@ body.page-template-template-demo {
 					width: 100%;
 					margin: 0 auto;
 
-					@media (max-width: $breakpoint-tablet - 1px) {
-						display: none;
-					}
+					// @media (max-width: $breakpoint-tablet - 1px) {
```
Comment (jremen) [inline assets/styles/layouts/_FullScreen.scss:126]: remove if no longer needed

**Review by jremen — COMMENTED** [review]
Fix some small issues – remove commented out code, badges save as PNG, try to optimize woman background

**Review by jremen — CHANGES_REQUESTED** [review]
@thujcak 
Trustpilot logo is missing…

**Review by jremen — APPROVED** [review]

**Review by mvstepanek — APPROVED** [review]

**Pull request merged**